models/nft/avalanche_c/nft_avalanche_c_schema.yml

```
@@ -15,3 +15,45 @@ models:
         description: "Aggregator project's contract address"
       - name: name
         description: "Aggregator project's name"
+  - name: nft_avalanche_c_transfers
+    meta:
+      blockchain: avalanche_c
+      sector: nft
+      contributors: hildobby
+    config:
+      tags: ['nft', 'avalanche_c', 'transfers']
+    description: >
+        NFT transfers
+    columns:
+      - name: block_time
+        description: "UTC event block time"
+      - name: block_number
```

You can add a uniqueness test here if you wish, currently the unique key only consists of one column `unique_transfer_id` but there may be a list of columns that form a unique key in the future.
```suggestion
    description: >
        NFT transfers
    tests:
      - dbt_utils.unique_combination_of_columns:
          combination_of_columns:
            - unique_transfer_id
    columns:
      - name: block_time
        description: "UTC event block time"
      - name: block_number
```